src/js/client/resend.js

```diff
@@ -0,0 +1,42 @@
+// provide a front-end to /api/user/verify/resend
+/* global fluid, jQuery */
+(function () {
+    "use strict";
+
+    fluid.registerNamespace("gpii.express.user.frontend.resend");
+
+    fluid.defaults("gpii.express.user.frontend.resend", {
+        gradeNames: ["gpii.schemas.client.errorAwareForm.clientSideValidation"],
+        schemaKey: "user-resend.json",
+        templates: {
+            initial: "resend-viewport",
+            error:   "common-error",
+            success: "common-success"
+        },
+        model: {
+            user: null
+        },
+        ajaxOptions: {
+            url:      "/api/user/verify/resend",
+            method:   "POST",
+            json:     true,
+            dataType: "json"
+        },
+        rules: {
+            modelToRequestPayload: {
+                "":    "notfound", // Required to clear out the default rules from `templateFormControl`
```

Comment:
I have used it myself, but have also frequently had to disable the defaults.  I will prepare a ticket and repo to split out the responsible `ajaxCapable` grade from `gpii-handlebars` and we can fix it, test it, and review it in more depth there.